networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/rpc/beaconchain/methods/BeaconBlocksByRangeMessageHandler.java

```diff
@@ -57,8 +57,8 @@ public void onIncomingMessage(
       return;
     }
     sendMatchingBlocks(message, callback)
-        .thenAccept(success -> callback.completeSuccessfully())
-        .exceptionally(
```

Comment:
Just wondering - this change isn't strictly necessary is it?  `thenAccept(..).exceptionally(...)` should still work?

Reply:
Exceptionally returns another future so technically it still has a return type that can be ignored. It looks like the errorprone check is ok with very simple things like:
```
.exceptionally(error -> {
  LOG.error("Oh no", error);
  return null;
});
```
But complains if you do anything more complex like this case does.  That's probably reasonable - as soon as you're doing more complex work to handle errors you might get another exception from the error handling code which would be silently ignored.